lib/rubocop/cop/rspec/message_spies.rb

```diff
@@ -12,17 +12,22 @@ module RSpec
       #
       #   # bad
       #   expect(foo).to receive(:bar)
+      #   subject
```

Comment:
I'm not convinced that the use of `subject` is the best example.
Harvesting the tickets related to the cop, I've seen an example with `do_something`.